src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+* [Use the OpenTelemetry Java agent:](#java-agent) Instead of using our pre-instrumentated demo app, you can use the OpenTelemetry Java agent to automatically monitor our demo app. You don't need to open the demo code and make any changes&mdash;just install and run the agent alongside the app.
+* [Instrument the demo app manually:](#manual-instrum) In this track, you'll roll up your sleeves and tinker with the engine of the car. This is the approach to take if you want to have the most control over what telemetry is reported and want to see details about how it's done. You'll manually insert instrumentation into our demo app to capture telemetry and you'll configure the SDK to send that data to New Relic.
+
+<Callout variant="tip">
+You have two choices for exporting data from your application to New Relic via OTLP:
+* Directly from your app
+* Via an OpenTelemetry collector
+
+This guide covers the first option. If you wish to export your data via a collector, check out this [documenation](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/opentelemetry-collector-basic/) for details.
+</Callout>
+## Requirements [#requirements]
+
+Before you get started, make sure you have the following:
+
+* A New Relic account: Sign up for a [free account](https://newrelic.com/signup) if you don't already have one.
+* Java 17 or higher: If you don't already have this, see [these download options](https://www.java.com/en/download/manual.jsp).
+
+## Run our pre-instrumented program [#pre-instrum]
+
+This is a great option if you want us do the instrumentation so you can quickly see what's it's like to send data to New Relic and view it in our UI.
+
+1. Clone the demo app and navigate to the following directory:
+    ```
+    git clone git@github.com:newrelic/newrelic-opentelemetry-examples.git
+    cd newrelic-opentelemetry-examples/getting-started-guides/java/Instrumented
+    ```
+2. In your terminal, set these two environment variables to send data to your New Relic account:
+    ```
+    export OTEL_EXPORTER_OTLP_HEADERS=api-key=INSERT_YOUR_NEW_RELIC_LICENSE_KEY
+    export OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317
+
+    ```
+    * Make sure to use your [ingest license key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key).
+    * If your account is based in the EU, set the endpoint to: https://otlp.eu01.nr-data.net:4317
+
+3. Set this environment variable to name the demo app:
+    ```
+    export OTEL_SERVICE_NAME=getting-started-java
+    ```
+4. In the `java/Instrumented/` directory, run the application:
+    ```
+    ./gradlew bootRun
+    ```
+5. Open a new terminal and exercise the application from the `Instrumented/java` directory by running this command:
```

Comment:
not sure what 'exercise' means here

Reply:
@zuluecho9 I switched it to this wording that is similar to the other section:

Open a new terminal, go to the `Instrumented/java` directory, and run the following command to generate some traffic from the application: